openapi-ts.config.ts

```diff
@@ -1,23 +1,30 @@
-import path, { format } from "path";
-import { defaultPlugins } from '@hey-api/openapi-ts';
+import { defineConfig } from "@hey-api/openapi-ts";
 
-export default {
+export default defineConfig({
   input: "./spec/kinde-mgmt-api-specs.yaml",
   output: {
-    path: "lib/api",
     format: "prettier",
+    path: "lib/api",
   },
-  services: {
-    name: "{{name}}",
-    
-  },
-   plugins: [
-    ...defaultPlugins,
-    '@hey-api/client-fetch',
+  plugins: [
+    {
+      baseUrl: "https://your_kinde_subdomain.kinde.com",
+      exportFromIndex: true,
+      name: "@hey-api/client-fetch",
+    },
+    {
+      identifierCase: "preserve",
+      name: "@hey-api/typescript",
+    },
     {
       asClass: true,
-      name: '@hey-api/schemas',
-      // type: 'json', 
+      name: "@hey-api/sdk",
+      serviceNameBuilder: "{{name}}",
+    },
+    {
+      exportFromIndex: true,
+      name: "@hey-api/schemas",
+      nameBuilder: (name) => `$${name}`,
```

Comment:
In the next release this will support a simple string too

```suggestion
      nameBuilder: "${{name}}",
```